lib/TypeMapper.php

```diff
@@ -162,6 +165,46 @@ public static function setDefaultResourceClass($class)
 
         return self::$defaultResourceClass = $class;
     }
+
+        /**
+     * @return string           The default Graph class
+     */
+    public static function getDefaultGraphClass()
+    {
+        return self::$defaultGraphClass;
+    }
+
+    /**
+     * Sets the default graph class
+     *
+     * @param  string $class The graph full class name (e.g. \MyCompany\Graph)
+     *
+     * @throws \InvalidArgumentException
+     * @return string           The default Graph class
+     */
+    public static function setDefaultGraphClass($class)
+    {
+        if (!is_string($class) or $class == null or $class == '') {
+            throw new \InvalidArgumentException(
+                "\$class should be a string and cannot be null or empty"
+            );
+        }
+
+        if (!class_exists($class)) {
+            throw new \InvalidArgumentException(
+                "Given class should be an existing class"
+            );
+        }
+
+        $ancestors = class_parents($class);
+        if (($class != 'EasyRdf\Graph') && (empty($ancestors) || !in_array('EasyRdf\Graph', $ancestors))) {
```

Comment:
I would use `0 == count($ancestors)` instead of `empty($ancestors)` here. It improves code understanding. Never tried it with arrays but [doc](https://www.php.net/manual/en/function.empty.php) says:

> A variable is considered empty if it does not exist or if its value equals false.

Reply:
Here I've copied the implementation of TypeMapper::setDefaultResourceClass().
Perhaps `count() == 0` can be used (but not in Yoda notation, not used anywhere in the whole code).
And, even better: the whole block of conditions can be moved in a dedicated private function, to be reused.